modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/ValidationDate.js

```diff
@@ -169,27 +169,25 @@ const ValidationDate = ({
 					</>
 				);
 			})}
-			{selectedValidation.name !== 'dateRange' && (
-				<Text
-					key="errorMessage"
-					label={Liferay.Language.get('error-message')}
-					name={`${name}_errorMessage`}
-					onBlur={onBlur}
-					onChange={(event) => {
-						dispatch({
-							payload: {
-								errorMessage: event.target.value,
-							},
-							type: EVENT_TYPES.CHANGE_ERROR_MESSAGE,
-						});
-					}}
-					placeholder={Liferay.Language.get('this-field-is-invalid')}
-					readOnly={readOnly}
-					required={false}
-					value={localizedValue(errorMessage)}
-					visible={visible}
-				/>
-			)}
+			<Text
+				key="errorMessage"
+				label={Liferay.Language.get('error-message')}
+				name={`${name}_errorMessage`}
+				onBlur={onBlur}
+				onChange={(event) => {
+					dispatch({
+						payload: {
+							errorMessage: event.target.value,
+						},
+						type: EVENT_TYPES.CHANGE_ERROR_MESSAGE,
+					});
+				}}
+				placeholder={Liferay.Language.get('this-field-is-invalid')}
+				readOnly={readOnly}
+				required={false}
+				value={localizedValue(errorMessage)}
+				visible={visible}
+			/>
```

Comment:
We better avoid use `ddm-form-field-types` into internal sidebar components in order to avoid extra renderizations and others overheads:
 ```suggestion
			<label htmlFor={`${name}_errorMessage`}>{Liferay.Language.get('error-message')}</label>
			<ClayInput
				id={`${name}_errorMessage`}
				name={`${name}_errorMessage`}
				onBlur={onBlur}
				onChange={(event) => {
					dispatch({
						payload: {
							errorMessage: event.target.value,
						},
						type: EVENT_TYPES.CHANGE_ERROR_MESSAGE,
					});
				}}
				placeholder={Liferay.Language.get('this-field-is-invalid')}
				value={localizedValue(errorMessage)}
			/>
```

**Note**: Since the exemple is doing `${name}_errorMessage` 3 times, may be a good idea extract it to a `const`